src/applications/third-party-apps/api/stub.js

```diff
@@ -0,0 +1,100 @@
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+        ],
+        privacyPolicyURL: '',
+        termsOfServiceURL: '',
+      },
+    },
+    {
+      id: '3',
+      type: 'third-party-app',
+      attributes: {
+        name: 'iBlueButton',
+        iconURL:
+          'https://www.apple.com/v/ios/health/c/images/overview/icon_health__c9yeoina5qaa_large_2x.png',
+        categories: ['Health'],
+        platforms: ['iOS', 'Android'],
+        appURL: 'https://www.ibluebutton.com/',
+        description:
+          'iBlueButton places your VA medical information securely onto your phone or tablet, assembled into an interactive health record, with personalized safety warnings for severe drug interactions, opioids, chronic condition care guidelines, and information regarding and risk for severe COVID-19 infection. iBlueButton has been a VA Blue Button health partner since 2012 and is available to Veterans free of charge.',
+        permissions: [
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+        ],
+        privacyPolicyURL:
+          'https://ice.ibluebutton.com/docs/ibb/privacy_policy.html',
+        termsOfServiceURL: 'https://ice.ibluebutton.com/docs/ibb/eula.html',
+      },
+    },
+    {
+      id: '4',
+      type: 'third-party-app',
+      attributes: {
+        name: 'MyLinks',
+        iconURL:
+          'https://www.apple.com/v/ios/health/c/images/overview/icon_health__c9yeoina5qaa_large_2x.png',
+        categories: ['Health'],
+        platforms: ['Web'],
+        appURL: 'https://www.mylinks.com/',
+        description:
+          'MyLinks is a free application that provides Veterans with their own secure and complete personal health record fully under their control. Veterans can gather, aggregate, and share their health records from their VA and non-VA health care providers into one place. They can add other important documents and images, connect devices, and keep a journal. MyLinks is accessible from any mobile device.',
+        permissions: [
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+          '[Placeholder copy - scopes from OAuth listed here]',
+        ],
+        privacyPolicyURL: 'https://app.mylinks.com/privacypolicy',
+        termsOfServiceURL: 'https://app.mylinks.com/termsofservice',
+      },
+    },
+  ],
+  meta: {
+    count: 5,
+  },
+};
```

Comment:
I'm a little wary of building on top of an ideal data structure since the real thing could be so different and have many limitations, but this looks good to me.